core/src/JsonLogic/IO/Operation.hs

```diff
@@ -23,23 +23,25 @@ where
 {- ORMOLU_ENABLE -}
 import qualified Data.Map as M
 import JsonLogic.IO.Mapping
-import JsonLogic.IO.Operation.Misc (log, miscOperations, trace)
+import JsonLogic.IO.Operation.Misc (miscOperations, trace, log)
 import JsonLogic.IO.Type
 import JsonLogic.Json
 import qualified JsonLogic.Operation as O
 import qualified Prelude as P
 
--- Default operators
+-- | A map of all the default operations.
 defaultOperations :: Operations
 defaultOperations = M.unions [arrayOperations, booleanOperations, dataOperations, miscOperations, numericOperations, stringOperations]
 
-arrayOperations, booleanOperations, dataOperations, numericOperations, stringOperations :: Operations
+-- | Groups of operations on similar data.
+arrayOperations, booleanOperations, dataOperations,  numericOperations, stringOperations :: Operations
```

Comment:
Why is there an extra space before `numericOperations`?

Reply:
```suggestion
arrayOperations, booleanOperations, dataOperations, numericOperations, stringOperations :: Operations
```